Tests/XCUITests/IntegrationTests.swift

```diff
@@ -68,7 +69,12 @@ class IntegrationTests: BaseTestCase {
     private func waitForInitialSyncComplete() {
         navigator.nowAt(BrowserTab)
         navigator.goto(SettingsScreen)
-        waitForExistence(app.tables.staticTexts["Sync Now"], timeout: 25)
+        waitForExistence(app.staticTexts["FIREFOX ACCOUNT"], timeout: TIMEOUT)
+        waitForNoExistence(app.staticTexts["Sync and Save Data"])
+        waitForExistence(app.tables.staticTexts["Sync Now"], timeout: TIMEOUT_LONG)
+        waitForExistence(app.tables.staticTexts["Syncing…"], timeout: TIMEOUT_LONG)
```

Comment:
I am not sure if this is going to work...if it apperas `Sync Now` text it does not appear `Syncing` and the other way around... 🤔

Reply:
Sometimes, `Syncing` appears when the settings screen is opened. The browser seems to poll the account frequently (say every 20 seconds). In this case, we wait for another round of "Syncing". 🤔 💭

Reply:
Let's ensure that `Syncing` appears at least once, not necessarily before `Sync Now`.

Reply:
I'm not very confident with this solution. We are waiting for existence for two different strings without doing something since one appears to get the other. If syncing appear sometimes, the other times the test will fail waiting for it to be shown.

That said, I open to try this and see if the tests are more stable